.circleci/config.yml

```diff
@@ -18,13 +13,102 @@ publish_with_tags_filter: &publish_with_tags_filter
     tags:
       only: /^v[0-9]+\.[0-9]+\.[0-9]+/
 
+parameters:
+  # used to trigger the workflow
+  trigger:
+    type: boolean
+    default: true
+
+  # parameter per package
+  ef-runtime-client:
+    type: boolean
+    default: false
+
+  ef-demo-component:
+    type: boolean
+    default: false
```

Comment:
I'm not used to parameters in circle ci, do we manually set those values when running the workflow?

Reply:
These are pipeline parameters. They are set when calling the CircleCI pipeline end point. They are false by default because they are triggered by the custom script.